AndroidSignalRJavaClientSample/app/build.gradle

```diff
@@ -0,0 +1,33 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.example.mikaelmengistu.androidsignalrjavaclientsample"
```

Comment:
Probably should remove your name

Reply:
You could probably use `com.microsoft.aspnet.signalr.samples.AndroidJavaClient`